tools/stats/print_test_stats.py

```diff
@@ -653,27 +665,20 @@ def __init__(self, name: str) -> None:
         self.test_suites: Dict[str, TestSuite] = dict()
 
     def append(self, test_case: TestCase, test_type: str, duplicated_tests_dict: DuplicatedDict) -> None:
-        is_multi_test = self.name == 'test_cpp_extensions_aot' or \
-            self.name == 'distributed/test_distributed_spawn' or \
-            self.name == 'distributed/test_c10d_gloo' or \
-            self.name == 'cpp'  # The caffe2 cpp tests spawn duplicate test cases as well.
-        if is_multi_test:
-            suite_name = test_case.class_name + '__' + test_type
```

Comment:
I didn't understand why this was added, and it doesn't seem crucial for the PR that introduced it. https://github.com/pytorch/pytorch/pull/60206/files Removing for clarity.